scripts/set-up-workload-identity-configs.sh

```diff
@@ -72,6 +72,12 @@ gcloud iam service-accounts add-iam-policy-binding --project="${GCP_PROJECT}" \
   --member="serviceAccount:${FLEET_HOST_PROJECT}.svc.id.goog[config-management-system/root-reconciler]" \
   "e2e-test-gcr-reader@${GCP_PROJECT}.iam.gserviceaccount.com"
 
+echo "Granting the e2e-metric-writer service account the 'roles/iam.workloadIdentityUser' role for the '${FLEET_HOST_PROJECT}' project"
+gcloud iam service-accounts add-iam-policy-binding --project="${GCP_PROJECT}" \
```

Comment:
Why do we need a binding to the fleet project? Are you intending to test fleet integration?

Reply:
I was adding it here for consistency, the test does not require fleet integration.

Reply:
I'm struggling to come up with a use case for this, I expect this isn't needed